docs/development/spi.md

```diff
@@ -2,12 +2,13 @@
 title: Service Provider Integration Deployment
 ---
 
-## Post-bootstrap Service Provider Integration(SPI) Configuration
+## SPI Deployment environment promotion
+
+SPI has 3 deployment [overlays](https://github.com/redhat-appstudio/infra-deployments/tree/main/components/spi/overlays) - `development`, `staging` and `production`. `development` and `staging` are updated together by autmatically created PR after each commit into `main` branch of [SPI Operator repository](https://github.com/redhat-appstudio/service-provider-integration-operator). To promote `staging` into `development`, manually create the PR. Typically update commit and image hashes of base `production` [kustomization.yaml](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/spi/overlays/production/base/kustomization.yaml) to match the `staging` [kustomization.yaml](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/spi/overlays/staging/base/kustomization.yaml).
```

Comment:
```suggestion
automatically
```

Reply:
And i suppose there must be a comma after `typically`

Reply:
fixed